releases/lib/common/context.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Context for the running server, providing injections for various services.
+class Context {
```

Comment:
Are you envisioning this as having the same roll as the Config.dart class has currently?

Reply:
Yes, but making it more as the way dependencies are injected. I still see the need for a `Config` service for having dynamic feature rollout.

Next PR we can add `context.getDatastore()` and `context.getGithub()`, which can be reusable in tests.